lib/check-self-update/fetch-module-registry-info.ts

```diff
@@ -12,12 +9,15 @@ export function fetchModuleRegistryInfo(name: string) {
     const url = [REGISTRY_URL, name, 'latest'].join('/');
 
     return new Promise((resolve, reject) => {
-        tiny.get({url}, (error, result: { body: any }) => {
+        request({ url, method: 'GET' }, (error, response, body) => {
             if (error) {
                 reject(error);
                 return;
             }
-            resolve(result.body);
+            if (body)
+                resolve(JSON.parse(body));
```

Comment:
@JamilOmar Use `{ json: true}` in the options then you don't need a manual parse.

Reply:
comment addressed